src/phantom.js

```diff
@@ -45,7 +36,20 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        if (typeof config.logger !== 'object') {
```

Comment:
Oh crap. Well sounds like we should move the default to `new Phantom()` then. I'll fix that my self since you have been so patient. :)